### ISSUE TITLE
Table structure

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#572]
Issue group: Tables
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 1.3.1: Info and Relationships
Needs design: 

### Description
The table on the Security keys page has four column headers: “label”, “value”, “action”, “”. The header for the last column is empty, meaning no heading information is provided for that column and provides no labelling/context for an AT user. It seems this table should have three columns: “label”, “key” and “action”. The “key” doesn’t appear in the table but shows up in the alert when you attempt to remove it; which implies that it should have been in the table data to identify the “key” you would like to remove. Alternatively, if the “key” is sensitive, the table should only contain the “label” and “action”, with the “label” being mentioned in the alert when removing it. 

![](https://dl.airtable.com/.attachments/05565f0e836c6a50bde3d646a4431ec3/a49352a3/Capturedecranle2022-03-15a09_52_50.png)

**Comment by adriannelee** [comment]
Few PRs for this and other a11y issues. @amazingphilippe will link the PRs and move them into Review

**Comment by amazingphilippe** [comment]
This PR removes the extra table column in the security key table https://github.com/cds-snc/notification-admin/pull/1734

**Comment by amazingphilippe** [comment]
@andrewleith will try to test security keys table locally
